test/functional/toolchains/env/test_virtualenv_powershell.py

```diff
@@ -213,6 +213,7 @@ class Pkg(ConanFile):
     assert "MYTOOL 1!!" in client.out
 
 
+@pytest.mark.skipif(platform.system() != "Windows", reason="Test for powershell")
```

Comment:
This test never runs powershell launchers, so it would be fine in any platform I guess. I am fine with running it only on Windows.

Reply:
sorry, I added that to the wrong test 🙄